src/ReleaseHistory.md

```diff
@@ -448,6 +448,7 @@
 * BUGFIX: Validation rule `SARIF1015` incorrectly required `originalUriBaseIds` to be contain URIs. https://github.com/microsoft/sarif-sdk/issues/1485
 * BUGFIX: Multitool transform mishandled dottedQuadFileVersion. https://github.com/microsoft/sarif-sdk/issues/1532
 * BUGFIX: Restore missing FxCop converter unit test. https://github.com/microsoft/sarif-sdk/issues/1575
+* BUGFIX: Multitool transform mishandled date/time values in property bags. https://github.com/microsoft/sarif-sdk/issues/1577
```

Comment:
>BUGFIX [](start = 2, length = 6)

It is clever to add bug fixes in ascending order of issue id. That's both helpful for locating a problem and also helps with merging this frequently touched file. I've taken a mental note to help maintain this as a standard, thanks for setting the pattern.